sdk/rust/oak_tests/src/lib.rs

```diff
@@ -100,61 +106,43 @@ const RETRY_INTERVAL: std::time::Duration = std::time::Duration::from_millis(800
 /// Convenience helper to build and run a single-Node Application with the given module name, using
```

Comment:
The comment still mentions the module name. Can you update it?

Reply:
Done.